app/models/user.rb

```diff
@@ -12,6 +13,30 @@ class User < ApplicationRecord
     length: { minimum: Settings.user.validates.pass_length_min }
   has_secure_password
 
+  def User.digest string
```

Comment:
```suggestion
  class << self
```
sử dụng cách này để khai báo class method nhé em